README.md

```diff
@@ -1,6 +1,7 @@
 # Version
 
 [![Build Status](https://travis-ci.com/nikolaposa/version.svg?branch=master)](https://travis-ci.com/nikolaposa/version)
+[![Build](https://github.com/nikolaposa/version/workflows/Build/badge.svg?branch=master)](https://github.com/nikolaposa/version/actions)
```

Comment:
Adding a badge here.

Note that `Build` in the URL refers to the name of the workflow. When the name of the workflow contained spaces, they would need to be URL-encoded.

For example:

```diff
diff --git a/.github/workflows/build.yaml b/.github/workflows/build.yaml
index 958cb47..aa1a9dc 100644
--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -1,6 +1,6 @@
 # https://docs.github.com/en/actions

-name: Build
+name: Continuous Integration

 on:
   pull_request: null
diff --git a/README.md b/README.md
index 7ed172e..f4d7ca8 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 # Version

 [![Build Status](https://travis-ci.com/nikolaposa/version.svg?branch=master)](https://travis-ci.com/nikolaposa/version)
-[![Build](https://github.com/nikolaposa/version/workflows/Build/badge.svg?branch=master)](https://github.com/nikolaposa/version/actions)
+[![Build](https://github.com/nikolaposa/version/workflows/Continous%20Integration/badge.svg?branch=master)](https://github.com/nikolaposa/version/actions)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/nikolaposa/version/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/nikolaposa/version/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/nikolaposa/version/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/nikolaposa/version/?branch=master)
 [![Latest Stable Version](https://poser.pugx.org/nikolaposa/version/v/stable)](https://packagist.org/packages/nikolaposa/version)
```